worker/raft/worker.go

```diff
@@ -26,6 +25,16 @@ const (
 	// defaultSnapshotRetention is the number of
 	// snapshots to retain on disk by default.
 	defaultSnapshotRetention = 2
+
+	// bootstrapAddress is the raft server address
+	// configured for the bootstrap node. This address
+	// will (must) be used at least until the cluster
+	// is grown beyond a single node.
+	//
+	// The .localhost label ensures that the address
+	// will not resolve globally. We do not require
+	// or expect the address to be resolvable at all.
+	bootstrapAddress raft.ServerAddress = "juju.localhost"
```

Comment:
do we want 'juju.localhost' or just 'localhost'?
I'm not sure why we want it to *not* be resolvable. Though I suppose once we're in HA we don't want any of the other nodes to try and use it.

Reply:
> I'm not sure why we want it to not be resolvable. Though I suppose once we're in HA we don't want any of the other nodes to try and use it.

Mostly so we don't accidentally resolve some externally-controlled name. I'll change it to localhost, that's fine.